portal/src/graphql/adminapi/RolesScreen.module.css

```diff
@@ -1,12 +1,19 @@
 .content {
-  @apply mt-11;
+  height: 100%;
```

Comment:
This element's parent is a flex container, therefore use flex's method to layout. Use `flex: 1 1 0px` here in desktop, and `flex: 1 0 auto` in mobile.